CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# DRAFT Contributing Guidelines
+
+Thank you for considering a contribution to the club-40 group project! To get you started, here are a few guidelines we would like you to follow:
+
+* [Issues and Bugs](#issue)
+* [Feature Requests](#feature)
+
+## <a name="question"></a> Got a Question?
+
+Please do not open issues for general support such as "how to install this plugin" as we want to keep this issue tracker
+for bug reports and feature requests. Instead, we suggest you use the discussion feature.
+
+## <a name="issue"></a> Found a Bug?
+
+If you find a bug in the code, you can help us by [submitting an issue](#submit-issue).
+Or, better, submit a [pull request](#submit-pr) with a fix readily available!
+
+## <a name="feature"></a> Missing a feature?
+
+You can request a new feature by [submitting an issue](#submit-issue) to the issue tracker.
+If you would like to implement a new feature, please consider the impact it would have on the codebase:
+
+* For a major impact affecting multiple files, adding wholly new topics and sections to the documentation, etc.:
+    please open an issue first, outlining your proposal so that it can be discussed.
+    This allows us to better coordinate our resources around your proposal and help you in crafting your contribution.
+
+* Small features can be put together and directly [submitted as a Pull Request](#submit-pr).
+
+## <a name="submit"></a> Submission Guidelines
+
+### <a name="submit-issue"></a> Submitting an Issue
+
+Before you submit a new issue, please search the issue tracker, maybe an issue relating to your problem already exists,
+and the discussion might inform you of fixes readily available.
+
+We want to fix all the issues as soon as possible, but before fixing a bug we need to reproduce and confirm it. Please provide as much information as possible in your initial post and be on the lookout for a response asking for additional information.
+
+You can file new issues by selecting from our
+[new issue templates]( TODO ) and filling out the issue template.
+
+### <a name="submit-pr"></a> Submitting a Pull Request (PR)
+
+Before you submit your Pull Request, follow these steps:
+
+1. Search [GitHub](TODO ) for an open or closed PR that relates to your submission.
+   You wouldn't want to duplicate existing efforts.
+
+1. Be sure that an issue describes the problem you're fixing, or documents the design for the feature you're adding.
+   Discussing the design upfront helps to ensure that we're ready to accept your work.
+
+1. [Fork](https://github.com/Club-40/group-project-1 ) and then clone your fork.
+
+1. Load in the clone of your fork via the plugin loader of your choice.
+
+1. In your cloned repository, make your changes in a new branch:
+
+    ```shell
+    git checkout -b my-branch master
+    ```
+
+1. Make your changes, including tests when appropriate. Please refer to [`test/README.md`](TODO ) for a guide.
```

Comment:
The list numbers need to be fixed (it repeats 1 for a couple of them)

Reply:
I think Github Markdown should auto convert the `1.` in groups into a list, though I am not sure if it will in this case with the extra line breaks in between.

Reply:
Good catch. It looks right in the preview too.